ImageD11/sinograms/dataset.py

```diff
@@ -47,9 +51,16 @@ def __init__(self,
                  dset = "dataset"):
         """ The things we need to know to process data """
 
+        
+        # it's better to determine the omegamotor, dtymotor and detector from the masterfile itself...
+        # however, that relies on realistic dataroot, analysisroot, etc values being passed to the init
+        # the default values for dataroot etc. will always give a broken masterfile path
```

Comment:
The default was downloading your data and working in the same folder as the dataset.